app/models/cluster_group.rb

```diff
@@ -356,12 +352,25 @@ def can_subsample?
     end
   end
 
+  # set the point count for a cluster_group and return value
+  def set_point_count!
+    points = self.concatenate_data_arrays('x', 'coordinates').count
+    self.update!(points: points)
+    self.points
+  end
+
   ##
   #
   # CLASS INSTANCE METHODS
   #
   ##
 
+  def self.set_all_point_counts!
+    self.all.each do |cluster|
```

Comment:
does this need to be batched?  I haven't tried ClusterGroup.all in prod

Reply:
There's only ~1500 or so in prod, so I'm not worried about that part.  But since it needs to be called on each cluster individually I couldn't think of any other way to do this.  Plus, I wanted to be able to call the single method in the migration and do it in the background so that it doesn't slow down the deployment or clog the job queue with 1500 jobs.  If you have a suggestion for a cleaner way of doing this, I'm all ears.